### ISSUE TITLE
Next version?

### ISSUE THREAD

**Issue by marcratombotsoa** [luchariman/serpent#1]
When can we expect next version? 💃

**Comment by LucHariman** [comment]
Indeed, this app deserves to be improved and updated to include some features such as multiplayer mode, persistence of the scores on the cloud, social login, and it shall be multi-platform. 😜
